src/main/kotlin/racingcar/ui/ResultView.kt

```diff
@@ -0,0 +1,42 @@
+package racingcar.ui
+
+import racingcar.domain.model.CarRacer
+
+class ResultView {
+
+    fun show(result: List<List<CarRacer>>) {
+        println("")
+        println("실행 결과")
+        result.forEachIndexed { index, gameResult ->
+            showGameResult(gameResult)
+            showWinners(result.lastIndex, index, gameResult)
+        }
+    }
+
+    private fun showWinners(
+        lastIndex: Int,
+        index: Int,
+        gameResult: List<CarRacer>
+    ) {
+        if (lastIndex != index) return
+        println("")
+
+        val winners: MutableList<String> = mutableListOf()
+        val max: Int = gameResult.maxOf { it.position }
+        gameResult.forEach {
+            if (it.position == max) winners.add(it.name)
+        }
+
+        println("${winners.joinToString(", ")}가 최종 우승했습니다.")
+    }
+
+    private fun showGameResult(carRacers: List<CarRacer>) {
+        carRacers.forEach { racer ->
+            var result = "${racer.name} : "
+            repeat(racer.position) { result += "-" }
+            println(result)
```

Comment:
view 를 분리하셨기 때문에 안심하고 마음껏 view 를 꾸미실 수 있었겠지요? 😄 👍 
아래와 같이 순서만 살짝 바꿔주면 코드상에서도 어떻게 화면단에 노출될지 힌트를 줄 수 있습니다.

```suggestion
        carRacers.forEach { racer ->
            var result = ""
            repeat(racer.position) { result += "-" }
            println("${racer.name} : $result")
```

Reply:
그렇군요! 코드 상에 출력에 힌트를 줄 수 있다는 부분에 공감됩니다. 
제안된 코드로 변경합니다!